Two Player Chess/main.py

```diff
@@ -0,0 +1,326 @@
+                                                move.pawn_promotion=False
+                                            elif event.key == p.K_n:
+                                                gs.makePawnPromotion(move,"N")
+                                                move.pawn_promotion=False
+                                            else:
+                                                gs.makePawnPromotion(move,"Q")
+                                                move.pawn_promotion=False
+                                            p.display.set_caption("ChessAI")
+                            
+                                
+                                
+                                    
+                                # argument to makemove is generated by the engine
+                            
+                                
+
+                                move_made = True
+                               
+                                sq_selected = () # reset user clicks
+                                player_clicks = []
+                            
+                            
+                                
+                                # reset the user clicks after making the move each time 
+                        if not move_made:
+                            player_clicks = [sq_selected]
+                        
+                            
+                        #gs.makeMove(move)
+                        # to make the move
+                            
+            elif event.type == p.KEYDOWN:
+                if event.key == p.K_z:
+                    gs.undoMove()
+
+                    move_made = True
+                    # when the user undoes a move the valid moves change
+                    # so change the flag variable to true
+
+                    # to update the valid moves
+        if move_made:
+            valid_moves = gs.getvalidmoves()
+            move_made = False
+    
+
+
+        # calling the draw boardand pieces fn
+        draw_game_state(screen,gs)
+        clock.tick(fps)
+        p.display.flip()
+        # to update the display
+
+# method to draw sqs on board and graphics of a current gamestate
+def draw_game_state(screen,gs):
+
+    # to draw squares on the board
+    drawboard(screen)
+
+    #board-->pieces order ofc matter karega nhi toh pieces piche chip jayenge 
+
+    # to draw pieces
+    drawpieces(screen,gs.board) # board from engine gamestate ka object gs , isliye dot
+
+def drawboard(screen):
+    # lets draw squares
+    # white and grey alternate
+    # make list to store white and grey switch karna easy hoga
+    # colors = [p.Color("white"), p.Color("dark gray")]
+    images = [p.image.load(r"C:\Users\MSHOME\Desktop\Newfolder\COC_Project_X_ChessAI\ChessAI\images\ltb.jpg").convert_alpha(),p.image.load(r"C:\Users\MSHOME\Desktop\Newfolder\COC_Project_X_ChessAI\ChessAI\images\dtb.jpg").convert_alpha()]
```

Comment:
path needs to be changed
```suggestion
    images = [p.image.load(r"C:\Users\MSHOME\Desktop\Newfolder\COC_Project_X_ChessAI\ChessAI\images\ltb.jpg").convert_alpha(),p.image.load(r"C:\Users\MSHOME\Desktop\Newfolder\COC_Project_X_ChessAI\ChessAI\images\dtb.jpg").convert_alpha()]
```